.github/workflows/no_pull_requests_to_main.yml

```diff
@@ -0,0 +1,18 @@
+# This is a basic workflow to thow an error if a pull request targets the main and historic branches
+
+name: No Pull Requests to Main and Historic Branches
+
+# Triggers the workflow pull request events but only for certain branches
+on:
+  pull_request:
+    branches: [ master, main, develop-historic ]
```

Comment:
I'd guess this list syntax probably works fine, but the examples shown here: 
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#on

Indicate it is more common to do: 
```
on:
    pull_request:
        branches:
            - master
            - main
            - 'develop-historic'
```
Quote may or may not be needed due to the '-' on the last one.